### ISSUE TITLE
Request to approve SSL cert followed by no process associated

### ISSUE THREAD

**Issue by GrahamH56** [dotnet/machinelearning-modelbuilder#2050]
**System Information (please complete the following information):**
 - Model Builder Version (available in Manage Extensions dialog): 16.13.1.2210302
 - Visual Studio Version : Microsoft Visual Studio Community 2022 (64-bit) - Current Version 17.1.0

**Describe the bug**
- On which step of the process did you run into an issue:
                                                            In consume your model after step 2 
- Clear description of the problem:
                                                             Request to approve ASP.net SSL cert (which I didn't do) Seemed to run with an expected result except 
                                                            " no process associated" was flagged.

**To Reproduce**
1. Changed code line to enable this to build.
This code had to be added manually 
                using myMLApp;
                // Add input data
                var sampleData = new SentimentModel.ModelInput()
                {
                    Col0 = "This restaurant was wonderful."
                };

This code line was changed to enable clean build. 
FROM:
string sentiment = result.Prediction == 1 ? "Positive" : "Negative";
TO :
string sentiment = result.PredictedLabel == 1 ? "Positive" : "Negative";
3. run program without debug
4.  Requested to approve ASP.net SSL cert (which I didn't do) 
5. Completed as expected but had a warning of  " no process associated" 

**Expected behavior**
Completed as expected but with issues being flagged and I had to correct automated produced code.

**Screenshots**
![image](https://user-images.githubusercontent.com/61938765/156013549-6c231efa-9a97-48e7-ae21-f523820db397.png)

![image](https://user-images.githubusercontent.com/61938765/156013830-ae50c373-450b-4e6a-804a-ee4740f8b2b0.png)

**Additional context**

**Comment by luisquintanilla** [comment]
This issue appears to be related to the first-run experience in .NET for ASP.NET applications. Closing this issue. @GrahamH56 please feel free to create a new issue if you have any questions or problems with Model Builder.
